package-safelist.json

```diff
@@ -36,6 +36,7 @@
   "vendor/dayjs/**/*.min.js",
   "vendor/monolog/monolog/src/*.php",
   "vendor/monolog/monolog/src/**/*.php",
+  "vendor/phpqrcode/**/*",
```

Comment:
needs to go in `.distfiles` too. Actually more important than here as we've moved to `pup`